.github/pull_request_template.md

```diff
@@ -1,16 +1,11 @@
-**Which issue is resolved by this Pull Request:**
-Resolves #
-
-<!-- You can delete this section if no corresponding issue -->
-
 **Description of your changes:**
 
 
 **Checklist:**
-- Do you want this PR cherry picked to release branch?
+- [ ] Do you want this PR cherry picked to release branch?
```

Comment:
Suggested revision:

Do you want this pull request (PR) cherry-picked into the current release branch?